lib/transit_data/glides_report/loader.ex

```diff
@@ -361,7 +361,7 @@ defmodule TransitData.GlidesReport.Loader do
   defp load_file_into_table(table_name, local_path) do
     local_path
     |> File.read!()
-    |> :erlang.binary_to_term([:safe])
+    |> :erlang.binary_to_term()
```

Comment:
More on this change: I found a way to check what new atoms were being created during this call, and it was just the keys of the maps. Nothing unexpected or risky.

I think by moving the module defs into a mix project, they now get loaded into the runtime only when one of their functions is first called.

So atoms like `:vehicle`, `:trip_id` etc don't yet exist during the S3-data-loading step since none of the modules that do the actual analysis have been loaded into the runtime yet.

This could maybe be solved by calling `Code.ensure_all_loaded([GlidesReport.TripUpdate, GlidesReport.VehiclePosition, ...])` but IMO it's safe to go with the simpler approach since we're working with trusted data.